packages/core/src/compiler/builtins/$richText/$richText.editor.tsx

```diff
@@ -272,7 +272,7 @@ function RichTextEditor(props: RichTextProps) {
 
       if (event.data.type === "@easyblocks-editor/rich-text-changed") {
         const { payload } = event.data;
-        const { editorContext } = (window.parent as any).editorWindowAPI;
+        const { editorContext } = (window as any).editorWindowAPI;
```

Comment:
`editorContext` is used heavily in this file and it should be the instance from the editor, not the copy in canvas.